src/components/Drawer/index.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { ReactElement } from 'react';
+import clsx from 'clsx';
+import { Flipped, Flipper } from 'react-flip-toolkit';
+import ClickAwayListener from '../../utilities/ClickAwayListener';
+import { Props } from './types';
+
+
+/**
+ * The Drawer component is used when you want to have content slide out from either the Top, Right, Bottom, or Left
+ */
+function Drawer({
+    anchorFrom,
+    children,
+    className,
+    isOpen = false,
+    ...otherProps
```

Comment:
You should probably pass through an `onClickAway` or similar prop to allow the scene to close the drawer.